src/Talis/Persona/Client/Base.php

```diff
@@ -55,40 +57,52 @@
     private $defaultTtl;
 
     /**
+     * @var string
+     */
+    private $appUserAgent;
+
+    /**
+     * @var string
+     */
+    private $phpVersion;
+
+    /**
      * Constructor
      *
      * @param array $config An array of options with the following keys: <pre>
      *      persona_host: (string) the persona host you'll be making requests to (e.g. 'http://localhost')
      *      persona_oauth_route: (string) the token api route to query ( e.g: '/oauth/tokens')
+     *      userAgent: Consuming application user agent string @since 2.0.0
+     *            examples: rl/5.2, rl, rl/5, rl/5.2 (php/5.3; linux/2.5)
```

Comment:
If you're ultimately going to end up putting these into New Relic for analysis, is it going to be a pain to get a different userAgent string for every release? If you started all the UA strings off with "rl" then I guess that's fine, you can query for it starting with a string rather than equalling a string.